src/storage-area.cpp

```diff
@@ -219,3 +219,22 @@ OrthancPluginErrorCode StorageRemoveCallback(const char *uuid, OrthancPluginCont
 
     return OrthancPluginErrorCode_Success;
 }
+
+OrthancPluginErrorCode OnStoredInstanceCallback(const OrthancPluginDicomInstance *instance, const char *instanceId) {
+    char msg[1024] = {0};
+    sprintf(msg, "OnStored Instance Id: %s", instanceId);
+    DEBUG_LOG(1,msg);
+
+    const void* instance_data = OrthancPluginGetInstanceData(globals::context, instance);
+    int size = OrthancPluginGetInstanceSize(globals::context, instance);
+    // update checksum - We need both md5 and size;
+    char* md5 = OrthancPluginComputeMd5(globals::context, instance_data, size);
```

Comment:
I think I've heard Brian say this returns the pre-filter md5 (probably a cached operation), so why don't we just pull in this <https://github.com/BCCF-UBCO-AD/Exercise1/blob/master/src/third-party/md5.cpp>